components/account/AccountHeader.vue

```diff
@@ -171,7 +183,15 @@ const personalNoteMaxLength = 2000
             <AccountLockIndicator v-if="account.locked" show-label />
             <AccountBotIndicator v-if="account.bot" show-label />
           </div>
-          <AccountHandle :account="account" overflow-unset line-clamp-unset />
+
+          <div flex items-center gap-1>
+            <AccountHandle :account="account" overflow-unset line-clamp-unset />
+            <CommonTooltip placement="bottom" :content="$t('account.copy_account_name')" no-auto-focus flex>
+              <button text-secondary-light text-sm i-ri:file-copy-line @click="copyAccountName">
```

Comment:
Would be great if the button had some sort of response, like changing the icon to a checkmark and changing back after a few seconds.

Reply:
Thank you for the review.🙂 That's a good suggestion. I modified it to show a check mark for 2 seconds.